src/sessions/SessionManager.ts

```diff
@@ -92,6 +94,11 @@ export class SessionManager {
         // Collect the user agent and domain
         this.collectAttributes();
 
+        // Set custom session attributes
+        if (this.config.customAttributesMap) {
+            this.setCustomAttributes(this.config.customAttributesMap);
+        }
```

Comment:
nit: The spread operator will ignore customAttributesMap if it is falsey.
```suggestion
        this.setCustomAttributes(this.config.customAttributesMap);
```

Reply:
Updated to use suggestion, thanks!